include/swift/AST/Attr.def

```diff
@@ -497,6 +497,17 @@ DECL_ATTR(_projectedValueProperty, ProjectedValueProperty,
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   89)
 
+SIMPLE_DECL_ATTR(_hasMissingDesignatedInits, HasMissingDesignatedInits,
+  OnClass | UserInaccessible | APIStableToAdd | ABIStableToAdd |
+  APIBreakingToRemove | ABIBreakingToRemove,
```

Comment:
I don't think it's APIBreakingToRemove, is it? ABI, maybe, since new things that depend on this won't backwards deploy, but…